src/nsls2api/tests/conftest.py

```diff
@@ -3,20 +3,46 @@
 import pytest_asyncio
 
 from nsls2api import models
+import secrets
 from nsls2api.infrastructure.config import get_settings
 from nsls2api.infrastructure.mongodb_setup import init_connection
 from nsls2api.models.beamlines import Beamline, ServiceAccounts
 from nsls2api.models.cycles import Cycle
 from nsls2api.models.facilities import Facility
 from nsls2api.models.proposal_types import ProposalType
 from nsls2api.models.proposals import Proposal
+from nsls2api.models.apikeys import ApiKey, ApiUser, ApiUserType
+from passlib.handlers.argon2 import argon2 as crypto
+from beanie import WriteRules
 
 
 @pytest_asyncio.fixture(scope="session", loop_scope="session", autouse=True)
 async def db():
     settings = get_settings()
     await init_connection(settings.mongodb_dsn)
 
+    TOKEN_BYTE_LENGTH = 32
```

Comment:
Just wondering if there was a reason that you couldn't use the `infrastructure.security.generate_api_key()` function?  Does it need something improving to make it work within a test ?  Or did you want to do it manually to avoid the dependency on an internal function ?